recipes/xppy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "xppy" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jsnowacki/xppy/archive/{{ version }}.tar.gz
+  sha256: 42df584090bc1a9d66d1faca6e4ec05ff50594577cbb4efcc46c25bae4cb8360
+  patches:
+    - 2_Add_method_xppy.set_cmd.diff
+    - 3-python3.patch
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - git
```

Comment:
Is git needed here?

Reply:
i'm really not sure.Lets see what happens when xppaut is merged in. I thought i was getting errors in patching.

I can't really reproduce locally.